app/components/Editor.js

```diff
@@ -95,13 +95,13 @@ const StyledEditor = styled(RichMarkdownEditor)`
 
   p {
     a {
-      color: ${props => props.theme.link};
-      border-bottom: 1px solid ${props => lighten(0.5, props.theme.link)};
+      color: ${props => props.theme.text};
```

Comment:
Unrelated, but noticed that the link colors outside of documents did not have accessible contrast in dark mode.